.claude/commands/ap-build.md

```
@@ -0,0 +1,47 @@
+
+Push-Location $root
+try {
+    cargo @arguments
+} finally {
+    Pop-Location
+}
+```
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)